### ISSUE TITLE
Issues with fullscreen

### ISSUE THREAD

**Issue by RavenMacDaddy** [nba-emu/NanoBoyAdvance#205]
First thing I noticed was that the - for many established ones - shortcuts do not work, being Alt+Enter and F11.

I didn't find any way to map this either, or any mentions in the config file, or any other wiki of any kind how to turn fullscreen on or off in the actual emulator.

---

The menu bar, at least for me, is always visible, though it's not interactable, and pressing buttons - once again, for many established ones - like Alt or Esc, didn't make a difference.

Basically, it's great if you can show or hide it in fullscreen, but having it both be visible at all times, combined with it not being interactable, really becomes a nuisance.

**Comment by RavenMacDaddy** [comment]
The shortcut does work in the nightly build, and the menu works there as well.

Is it possible that later builds will give the user the option to map the fullscreen toggle to another button (combination)?

**Comment by RavenMacDaddy** [comment]
Great, and also in case you didn't catch my edits I've now made some changes to my previous comment.

**Comment by fleroviux** [comment]
In the latest dev build the menu bar is now hidden by default in fullscreen mode (when a game is loaded).
I've also made it even simpler to exit fullscreen (via double-click on the screen or ESC key).

I don't have plans to implement shortcut remapping anytime soon, and I'm not sure how useful it really is. If there's a real demand, it should be requested as a separate Github issue, as it's not related to this issue per se.
